### DISCUSSION TITLE
After tournament has run, add games to snake Recent Games list

### DISCUSSION THREAD

**Discussion by mawilson** [BattlesnakeOfficial/feedback#207] [Feature Requests]
Currently tournament games are not added to a battlesnake page's Recent Games list, to avoid spoilers. If possible, it would be nice to have these games added to the list once the tournament has completed, so that we can rewatch them, add them as favorites, use them for test cases, etc.

**Comment by bvanvugt** [comment]
Yeah this has come up in a couple different places, we may have released something recently to address it. I'll check with the team and get back to you 👍

**Comment by chris-bsnake** [comment]
We have deployed a fix.  Now once a tournament is completed the tournament games should show up in the Recent Games list for your snake
